app/src/lib/git/core.ts

```diff
@@ -247,6 +247,12 @@ function getDescriptionForError(error: DugiteError): string {
       return 'The push was rejected by the remote server because a required status check has not been satisfied.'
     case DugiteError.BranchRenameFailed:
       return 'The branch could not be renamed.'
+    case DugiteError.PathDoesNotExist:
+      return 'The path does not exist on disk.'
+    case DugiteError.InvalidObjectName:
+      return 'The object was not found in the Git repository.'
```

Comment:
I feel like this error message wouldn’t help a Git beginner figure out what’s wrong. Is there a way to communicate what a object is in Git terms so new users will understand?